src/Mvc/Mvc.ViewFeatures/src/Microsoft.AspNetCore.Mvc.ViewFeatures.csproj

```diff
@@ -25,6 +25,7 @@
     <Reference Include="Microsoft.AspNetCore.Antiforgery" />
     <Reference Include="Microsoft.AspNetCore.Diagnostics.Abstractions" />
     <Reference Include="Microsoft.AspNetCore.Html.Abstractions" />
+    <Reference Include="Microsoft.AspNetCore.Http.Results" />
```

Comment:
Adding this dependency so that it's possible to use `IResultExtensions` and implement support for

```cs
app.MapGet("/counter", () => Results.Extensions.RazorComponent<Counter>(new { IncrementAmount = 10 }));
```

However I'm not at all sure it's really desirable to put built-in framework things on `Results.Extensions`. I've only done it this way for now because we don't yet have any consensus on the right place to put this, or even if we should just force people to do the following instead:

```cs
app.MapGet("/counter", () => new RazorComponentResult<Counter>(new { IncrementAmount = 10 }));
```

... which is fewer characters to type and offers more flexibility, but is different to the common patterns for other result types.

Reply:
Update: that was weird so I removed it.